.rubocop_todo.yml

```diff
@@ -26,6 +26,28 @@ RSpec/LetSetup:
     - 'spec/jobs/start_training_mail_job_spec.rb'
     - 'spec/jobs/test_bulk_mail_job_spec.rb'
 
+# Offense count: 1
+# Configuration parameters: AllowedPatterns.
+# AllowedPatterns: ^expect_, ^assert_
+RSpec/NoExpectationExample:
+  Exclude:
+    - 'spec/support/shared/email_prompt.rb'
+
+# Offense count: 12
+# This cop supports unsafe autocorrection (--autocorrect-all).
+RSpec/ReceiveMessages:
+  Exclude:
+    - 'spec/controllers/users/omniauth_callbacks_controller_spec.rb'
+    - 'spec/helpers/link_helper_spec.rb'
+
+# Offense count: 1
+# This cop supports unsafe autocorrection (--autocorrect-all).
+# Configuration parameters: EnforcedStyle.
+# SupportedStyles: constant, string
+RSpec/VerifiedDoubleReference:
+  Exclude:
+    - 'spec/services/gov_one_auth_service_spec.rb'
+
```

Comment:
How do you feel about these additions @martikat? Can they be made to conform without causing a headache?